cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,10 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Data.ReadInstanceData()
```

Comment:
So this confuses me. Aside the fact that I question the following print statement, why would I need to read the data at this point. Is it not initialized and read in _root.go_?